### DISCUSSION TITLE
Unable to Access The Student Developer Pack

### DISCUSSION THREAD

**Discussion by akshan39** [community/community#56232] [GitHub Education]
### Hi everyone,

Hello everyone,
I just signed up for the global student program as a student. The website says that I have been approved as a student but I get redirected to verification as a student every time I try to sign up fro the student developer pack. Can anyone hep me out to get access to my student developer pack? FYI, I tried re-verifying my details, but the website says that I'm already verified.

**Comment by Ntshangase** [comment]
Hello @akshan39 

It looks like your application has been approved, you an check your application status here: [Click me](https://education.github.com/discount_requests/application)

**Comment by samus-aran** [comment]
Hey @akshan39  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
